pkg/services/serviceaccounts/manager/service.go

```diff
@@ -39,6 +39,28 @@ func ProvideServiceAccountsService(
 	}
 	serviceaccountsAPI := api.NewServiceAccountsAPI(s, ac, routeRegister)
 	serviceaccountsAPI.RegisterAPIEndpoints(cfg)
+
+	basicKeys := store.GetNonServiceAccountAPIKeys(context.Background())
+	if len(basicKeys) > 0 {
+		s.log.Debug("Launching background thread to upgrade API keys to service accounts", "numberKeys", len(basicKeys))
```

Comment:
```suggestion
		s.log.Info("Launching background thread to upgrade API keys to service accounts", "numberKeys", len(basicKeys))
```
potential: change this to info